src/ray/gcs/gcs_client/gcs_client.h

```diff
@@ -77,18 +55,6 @@ class GcsClientOptions {
   // Gcs address
   std::string gcs_address_;
   int gcs_port_ = 0;
-
-  // redis server address
-  std::string redis_ip_;
-  int redis_port_ = 0;
-
-  // Password of GCS server.
-  std::string password_;
-
-  // Whether to enable connection for contexts.
-  bool enable_sync_conn_{true};
-  bool enable_async_conn_{true};
-  bool enable_subscribe_conn_{true};
```

Comment:
We can delete these options in `RedisClientOptions` as well.

Reply:
I'll update it in the next PR.